python/cudf/cudf/io/orc.py

```diff
@@ -339,7 +342,10 @@ def read_orc(
             path_or_data=source,
             compression=None,
             use_python_file_object=use_python_file_object,
-            **kwargs,
+            storage_options=storage_options,
+            bytes_per_thread=256_000_000
+            if bytes_per_thread is None
+            else bytes_per_thread,
```

Comment:
> if bytes_per_thread is None:
    bytes_per_thread = 256_000_000

done 👍 



> Alternatively, should we specify `bytes_per_thread=256_000_000` in the function's default parameters?

I don't think so, because `bytes_per_thread` can only be passed if `use_python_file_object=False`, we have an active check for this something like:
```python
if use_python_file_object and bytes_per_thread is not None:
        raise ValueError(
            "bytes_per_thread is only supported when "
            "`use_python_file_object=False`"
        )
```